keras_eval/eval.py

```diff
@@ -145,6 +149,32 @@ def evaluate(self, data_dir=None, top_k=1, filter_indices=None, confusion_matrix
 
         return self.probabilities, self.labels
 
+    def compute_inference_probabilities(self, concept_dictionary, probabilities):
+        '''
+        Sets probabilities if model concepts are to be combined in test
```

Comment:
Can we make this description more clear? 

Something like:
> Apply probability inference on concepts defined in `concept_dictionary` and save them in `self.probabilities`

Reply:
I am changing it to "Combines probabilities based on key "group" in concept_dictionary and saves the values in self.probabilities"